### ISSUE TITLE
INFO is not valid prometheus type

### ISSUE THREAD

**Issue by ikitommi** [gnarroway/fumi#7]
https://prometheus.io/docs/concepts/metric_types/

**Comment by gnarroway** [comment]
Hi, could you please provide a bit more context? are you seeing an INFO type being output somewhere or are you referring to some comments in the code base? what version are you referring to?

There is an Info in the [underlying java client](https://github.com/prometheus/client_java/blob/main/simpleclient/src/main/java/io/prometheus/client/Info.java), (used eg for the [version info default exports](https://github.com/prometheus/client_java/blob/main/simpleclient_hotspot/src/main/java/io/prometheus/client/hotspot/VersionInfoExports.java)) but it is correctly output as a gauge:

```
# HELP jvm VM version info
# TYPE jvm gauge
jvm_info{runtime=\"OpenJDK Runtime Environment\",vendor=\"Azul Systems, Inc.\",version=\"11.0.11+9-LTS\"} 1.0
```

I think a previous version of this library may have output it as INFO, but without more details I can only guess what you are talking about.

**Comment by ikitommi** [comment]
sure, [here](https://github.com/gnarroway/fumi/blob/master/src/fumi/client.clj#L332-L342) fumi is registering `:info` types, which causes Prometheus 2.26.2 to fail with `invalid metric type "info"`.

**Comment by ikitommi** [comment]
ugh, that's not real library code. and this is fixed in 0.3.1. sorry for the noice.

**Comment by gnarroway** [comment]
no worries :) was trying to figure out how you were using the comment block. thanks for trying it out!
